typescript/packages/well-completions-plot/src/WellCompletionsPlot.tsx

```diff
@@ -1,80 +1,80 @@
-import { styled } from "@mui/material/styles";
-import React, { useMemo } from "react";
+import React from "react";
 import { useResizeDetector } from "react-resize-detector";
-import { PlotData } from "../../utils/dataUtil";
-import { TooltipProvider } from "../Common/TooltipProvider";
-import CompletionsPlot from "./CompletionsPlot";
-import { getLayout, Padding } from "./plotUtil";
-import StratigraphyPlot from "./StratigraphyPlot";
-import WellsPlot from "./WellsPlot";
+import { TooltipProvider } from "./components/TooltipProvider";
+import { CompletionsPlot } from "./components/CompletionsPlot";
+import { createLayout, Padding } from "./types/layoutTypes";
+import { StratigraphyPlot } from "./components/StratigraphyPlot";
+import { WellsPlot } from "./components/WellsPlot";
 
-const Root = styled("div")(({ theme }) => ({
-    display: "flex",
-    flex: 1,
-    height: "80%",
-    padding: theme.spacing(1),
-}));
+import { PlotData } from "./types/dataTypes";
+export { PlotData } from "./types/dataTypes";
 
-interface Props {
+import "./WellCompletionsPlot.css";
+
+export interface WellCompletionsPlotProps {
+    id: string;
     timeSteps: string[];
     plotData: PlotData;
 }
 
 const padding: Padding = { left: 80, right: 50, top: 70, bottom: 50 };
 /* eslint-disable react/display-name */
 /* eslint-disable react/prop-types */
-const WellCompletionsPlot: React.FC<Props> = React.memo(
-    ({ timeSteps, plotData }) => {
+export const WellCompletionsPlot: React.FC<WellCompletionsPlotProps> =
+    React.memo((props: WellCompletionsPlotProps) => {
         const { width, height, ref } = useResizeDetector({
             refreshMode: "debounce",
             refreshRate: 50,
             refreshOptions: { trailing: true },
         });
-        const layout = useMemo(
+
+        const layout = React.useMemo(
```

Comment:
Consider if `useMemo` is necessary here. The computation in `createLayout` does not seem very heavy.